app/experimenter/nimbus-ui/src/lib/helpers.tsx

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import {
+  createHistory,
+  createMemorySource,
+  LocationProvider,
+} from "@reach/router";
+import { render } from "@testing-library/react";
+
+export function renderWithRouter(
+  ui: React.ReactElement,
+  { route = "/", history = createHistory(createMemorySource(route)) } = {},
+) {
+  return {
+    ...render(<LocationProvider {...{ history }}>{ui}</LocationProvider>),
+    history,
+  };
+}
```

Comment:
Is this intended to be a catch-all helper file? Or do you think we'll have other render or router related stuff that would fit under this category to go here? Since this abstracts Router does this belong in `services/`? File organization is hard for me 😖

Also, I suppose this needs a `helpers.test.tsx` for complete coverage.

Reply:
I'm not sure, it's certainly a _test_ helper file at the moment. Maybe we can decide what to do with it when we need some other helper?

I think because I'm using this in tests it doesn't need a test file, or at least, Jest is telling me it's at 100% coverage. 🤔

Reply:
Les had a [similar thought](https://github.com/mozilla/experimenter/pull/3791#discussion_r510365204), maybe we should have a tests folder? Or at least explicitly exclude them in the jest config?

Reply:
Yeah, if this is for tests, I'd name it `test-helpers.tsx` or something so as not to attract other kinds of rando helpers.

Reply:
Done, went with `test-utils.tsx`. 🙂